app/Modules/Bookings/BookingsServiceInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Do we need an interface for this? Sounds like there's only one way this service will be implemented.

Reply:
I originally did this as an interface to bind it in the service container, maybe a singleton would be better? What are your thoughts on this?